samples/highcharts/series/legend-color/demo.details

```diff
@@ -0,0 +1,6 @@
+---
+ name: LegendColor option demo
+ authors:
+   - Andrzej Bułeczka
```

Comment:
We have assumed that we do not use Polish letters in this case; honestly, I do not know if this generates any problems in the pipeline, but this is the recommendation I received some time ago.